hut-finder-utilities/scripts/README.md

```diff
@@ -0,0 +1,85 @@
+# Database Setup Guide
+This guide runs you through the setup process of creating a database.
+This assumes you have `postgres` installed.
```

Comment:
A cheeky docker image would be really cool to simplify the setup and keep everything containerised.